src/components/bookItem.js

```diff
@@ -10,22 +11,25 @@ const Book = ({
   const handleDelete = () => onDelete(id);
 
   return (
-    <div className="book-item">
-      <h2>{title}</h2>
-      <p>
-        Author:
-        {author}
-      </p>
-      {category && (
-      <p>
-        Category:
-        {category}
-      </p>
-      )}
-      <button type="button" onClick={handleDelete}>
-        Delete
-      </button>
-    </div>
+    <>
+      <div className="book-item">
+        <div>
+          {category && <p className="category-head">{category}</p>}
+          <h2 className="book-title">{title}</h2>
+          <p className="author-name">{author}</p>
+          <div>
+            <button className="comments-btn" type="button">Comments</button>
+            <button className="remove-btn" type="button" onClick={handleDelete}>
+              Remove
+            </button>
+            <button className="edit-btn" type="button">Edit</button>
+          </div>
+        </div>
+        <div>
+          <Progress />
+        </div>
+      </div>
```

Comment:
- [ ] Please fix your overlap text from this portion and stick to the [Zaplin](https://zeplin.io/) design :+1: check below :point_down: the comparison between your design and [Zaplin](https://zeplin.io/) design :slightly_smiling_face:

**Your Design**
![Screenshot from 2023-06-25 03-17-22](https://github.com/MoyasiGinko/React-BookStore/assets/45226848/5cf74e6a-f89f-4a6c-9af5-970c0f313df1)

**Zaplin Design**
![Screenshot from 2023-06-25 03-21-51](https://github.com/MoyasiGinko/React-BookStore/assets/45226848/c433c9aa-dd03-4760-8723-bfb548056ae1)